### PR TITLE
[netapp-exporter] add inode alerts for manila shares

### PR DESCRIPTION
**Pull request by chuan137**

### CODE REVIEW

**Review by Carthaca — CHANGES_REQUESTED** [review]

**Review by Carthaca — APPROVED** [review]

**Comment by Carthaca** [comment]
I suggest to wait with the rollout for the playbook

File: prometheus-exporters/netapp-exporter/charts/netapp-cap-exporter/alerts/netapp-capacity.alerts [hunk 1]
```diff
@@ -99,6 +99,34 @@ groups:
...
+            tier: os
+            service: manila
+            context: netapp-volume-inode-usage
+            support_component: manila_netapp
+            playbook: 'docs/support/playbook/manila/share_inode_usage'
+        annotations:
+            summary: 'inode usage above 85%'
+            description: 'inode usage of Manila share {{ $labels.share_id }} is {{ $value }}'
+
+      - alert: NetappVolumeInodeUsageVeryHigh
```
[the remark this answers is not in the document]
Comment (geisslet) [reply prometheus-exporters/netapp-exporter/charts/netapp-cap-exporter/alerts/netapp-capacity.alerts]: pls check the note on the [playbook of the critical](https://github.wdf.sap.corp/monsoon/convergedcloud-opsdocu-ui/pull/1152#pullrequestreview-6596826)

File: prometheus-exporters/netapp-exporter/charts/netapp-cap-exporter/alerts/netapp-capacity.alerts [hunk 2]
```diff
@@ -99,6 +99,34 @@ groups:
...
+            playbook: 'docs/support/playbook/manila/share_inode_usage'
+        annotations:
+            summary: 'inode usage above 85%'
+            description: 'inode usage of Manila share {{ $labels.share_id }} is {{ $value }}'
+
+      - alert: ManilaShareInodeUsageVeryHigh
+        expr: max_over_time(netapp_volume_inode_files_used_percentage{app="netapp-capacity-exporter-manila"}[1h]) > 0.95
+        for: 1h
+        labels:
+            severity: critical
```
Comment (Carthaca) [inline prometheus-exporters/netapp-exporter/charts/netapp-cap-exporter/alerts/netapp-capacity.alerts:120]: let's set this down to `warning`, we have no proper 24/7 response for this anyway

**Review by geisslet — APPROVED** [review]
lgtm

**Pull request merged**